packages/commerce-sdk-react/README.md

```diff
@@ -404,6 +404,26 @@ useEncUserId() => {encUserId: String, getEncUserIdWhenReady: Promise}
 useUsid() => {usid: String, getUsidWhenReady: Promise}
 ```
 
+### `useStorageHelper()`
+
+This hook exposes helper functions to interact with data stored in local/cookie/memory storage
+
+Currently, the available helpers are:
+ * clearStorage: clears all data that the commerce-sdk-react sets in local/cookie storage
+
+```jsx
+import {useStorageHelper} from '@salesforce/commerce-sdk-react'
+
+const Example = (props) => {
+    // Typescript IDE intellisense for available options
+    const {clearStorage} = useStorageHelper()
+
+    clearStorage()
+
+    return (...)
+}
```

Comment:
We should use a real use case for the example, I'm just slightly worried that this isn't something useful after we ship it. so a good real use case warrants it

Reply:
This hook must not run on its own, like if you clear storage and do nothing else, I think your app is broken because all SCAPI requests will fail. I think we need some documentation on what else do you need to do to keep your app working

Reply:
Good callout, I'll update the README to highlight a better use case. The use case this was designed for is to allow a developer to reset a user session.

You can do this for guest users by calling `clearStorage()` to clear the session and then calling `loginGuestUser()` afterwards. For registered users, the `logout()` helper does exactly this after logging the registered user out. We could technically call `logout()` for guest users as it will reset the storefront in the same way, but you'll get a `400` error and the docs advise against this for the `logoutCustomer` endpoint: 

_[This should be called for Registered users that have logged in using SLAS. his should be called for registered users that have logged in using SLAS. This endpoint is not for use with guest users.](https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-login?)_

Reply:
What you wrote above definitely worth being added to the jsdoc for that hook

Reply:
I'm trying to step away from the implementation detail and think this problem from the high level, would there be a solution where the `logout` conditionally behaves differently for guest and registered user and replace the need for a `useStorageHelper` hook?

Reply:
Thought about this some more, I think there's a better way to do this